PavloKandur/HW7/task3.py

```diff
@@ -0,0 +1,16 @@
+def number_of_chars_in_word(word:str)->dict:
```

Comment:
good job
look at this 
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result